### ISSUE TITLE
Manifest payload naming standard

### ISSUE THREAD

**Issue by Sveino** [Haigutus/Energy-Reference-Data#39]
CIM in general shall not include any naming standard. The main idea to include manifest and DCAT is to avoid implementation to rely  on naming standard. However, as part of testing and where user interaction on the technical level is needed there is a need for a naming recommendation.
The shall follow the same logic that cim:IdentifiedObject.mRID and cim:IdentifiedObject.name where mRID is the machine interpreted identification and name is the user "identification".
A naming standard can also be useful for simple file based archiving tools that is based on the file name.

The updated naming standard need to cover the needs from CGMES profiles in CGM and TYNDP process in addition to the CSA, CCC, OPC and STA processes.

The current CGM names standard is document in: [Quality of CGMES datasets and calculations v3.3](https://eepublicdownloads.azureedge.net/clean-documents/digital/QualityOfCGMESdatasetsAndCalculations_v3_3.pdf)
 3.2 FILE NAME AND FILE HEADER
The following mask is to be used to have a valid file name:
(snip)
<img width="363" alt="image" src="https://github.com/Haigutus/Energy-Reference-Data/assets/15873106/e2b31b01-2f1d-4d2a-ac5f-6007fc85773e">

The item in the naming standard need to be found in the header so that tools can generate it based on an information model and that is consistent with the content of the payload.

**Comment by Haigutus** [comment]
I would propose a rule, that filename can contain only data that can be extracted from file header.

Reasoning:
1. Currently some metadata is added to filename, that is not present inside the file and then the filename parsing becomes mandatory process. To avoid this in future we should force the rule and if additional metadata is needed, then first file header/manifest needs to be extended 

2. Filename can be automatically created at the moment of storage by extracting relevant metadata from the file header

**Comment by Sveino** [comment]
@Haigutus Yes, definitely - I was hoping this would come clear out of the text above. In the discussion with CSA it is clear that we need to have a name - may above proposal is based on this. Making sure that we can cover the current requirement. The next step would be to come up with a proposal that used our current header data.

**Comment by Sveino** [comment]
Updated above that the _[dcat:version] is referring to dcat:Dataset and not dcat:Distribution. It now refers to when a dataset is replaced by a new version with the same metadata, e.g. start and end validitiy period. dcat:version will follow semantic versioning, e.g. https://semver.org/.
